Applications/MaterialAssist/Client/MaterialAssistClientBoards.cs

```diff
@@ -0,0 +1,196 @@
+        private const string _Width = "width";
+        private const string _Comments = "comments";
+        private const string _RemovalType = "removalType";
+        private const string _Quantity = "quantity";
+
+        #endregion Constants
+
+        #region Read
+
+        /// <inheritdoc />
+        public async Task<IEnumerable<BoardEntity>> GetBoardEntities(int take, int skip = 0)
+        {
+            var url = $"{_BaseRoute}?take={take}&skip={skip}";
+
+            return await RequestEnumerable<BoardEntity>(new Uri(url, UriKind.Relative));
+        }
+
+        /// <inheritdoc />
+        public async Task<BoardEntity> GetBoardEntityById(string id)
+        {
+            var url = $"{_BaseRoute}?{_Id}={Uri.EscapeDataString(id)}";
+
+            return await RequestObject<BoardEntity>(new Uri(url, UriKind.Relative));
+        }
+
+        /// <inheritdoc />
+        public async Task<IEnumerable<BoardEntity>> GetBoardEntitiesByIds(IEnumerable<string> ids)
+        {
+            var url = $"{_BaseRoute}";
+
+            var boardCodes = new StringBuilder("?");
+            boardCodes.Append(string.Join("&", ids.Select(id => $"{_Id}={Uri.EscapeDataString(id)}")));
+
+            url += boardCodes;
+
+            return await RequestEnumerable<BoardEntity>(new Uri(url, UriKind.Relative));
+        }
+
+        /// <inheritdoc />
+        public async Task<IEnumerable<BoardEntity>> GetBoardEntitiesByBoardCode(string boardCode)
+        {
+            var url = $"{_BaseRoute}?{_BoardCode}={Uri.EscapeDataString(boardCode)}";
+
+            return await RequestEnumerable<BoardEntity>(new Uri(url, UriKind.Relative));
+        }
+
+        /// <inheritdoc />
+        public async Task<IEnumerable<BoardEntity>> GetBoardEntitiesByBoardCodes(IEnumerable<string> boardCodes)
+        {
+            var url = $"{_BaseRoute}";
+
+            var appendedBoardCodes = new StringBuilder("?");
+            appendedBoardCodes.Append(string.Join("&", boardCodes.Select(boardCode => $"{_BoardCode}={Uri.EscapeDataString(boardCode)}")));
+
+            url += boardCodes;
+
+            return await RequestEnumerable<BoardEntity>(new Uri(url, UriKind.Relative));
+        }
+
+        /// <inheritdoc />
+        public async Task<IEnumerable<StorageLocation>> GetStorageLocations()
+        {
+            var url = $"{_BaseRoute}/storageLocations";
+
+            return await RequestEnumerable<StorageLocation>(new Uri(url, UriKind.Relative));
+        }
+
+        #endregion Read
+
+        #region Create
+
+        /// <inheritdoc />
+        public async Task<BoardEntity> CreateBoardEntity(string boardTypeCode, BoardEntity boardEntity)
+        {
+            var url =
+                $"{_BaseRoute}?{_BoardTypeCode}={Uri.EscapeDataString(boardTypeCode)}";
+
+            return await PostObject(new Uri(url, UriKind.Relative), boardEntity);
+        }
+
+        /// <inheritdoc />
+        public async Task CreateBoardType(BoardType boardType)
```

Comment:
Looks different in materialManager. There a request is send.

Reply:
We adapt it to what Daniel already implemented for materialManager